lib/shopify-cli/commands/create/project.rb

```diff
@@ -24,7 +23,7 @@ def call(args, _name)
             @ctx,
             org_id: form.organization_id,
             title: form.title,
-            app_url: form.app_url,
+            app_url: 'https://shopify.github.io/shopify-app-cli/getting-started',
```

Comment:
We should probably make this page!

Reply:
I did! It is in this PR. I have also enabled github pages on this repo, so once this merges that link will be live

Reply:
LOL i missed it at the top there great good job